enterprise-registry/build-support/index.md

```diff
@@ -11,39 +11,28 @@ weight: 5
 
 CoreOS Enterprise Registry supports building Dockerfiles using a set of worker nodes. Build triggers, such as GitHub webhooks ([Setup Instructions]({{site.url}}/docs/enterprise-registry/github-build)), can be configured to automatically build new versions of your repositories when new code is committed. This document will walk you through enabling the feature flag and setting up multiple build workers to enable this feature.
 
-<img src="workers.png" class="img-center" alt="Enterprise Registry Build Workers"/>
-
 *Note:* This feature is currently in *beta*, so it may encounter issues every so often. Please report
 any issues encountered to support so we can fix it ASAP.
 
-## Enable the Feature Flag
-
-By default, the feature flag to enable build workers is off. In the Enterprise Registry `config.yaml`, change the following to enable build support:
-
-```yaml
-FEATURE_BUILD_SUPPORT: false
-```
-
-to
+## Visit the Management Panel
 
-```yaml
-FEATURE_BUILD_SUPPORT: true
-```
-
-### Update and Restart Enterprise Registry
+Sign in to a super user account and visit `http://yourregister/superuser` to view the management panel panel:
 
-If you're running an older version of the Enterprise Registry container, pull down a new copy:
+<img src="superuser.png" class="img-center" alt="Enterprise Registry Management Panel"/>
 
-```sh
-docker pull quay.io/coreos/registry
-```
+## Enable Building
 
-During the Enterprise Registry set up process, you were provided credentials to access this image. If your server has these configured, they will be used. If you get an authentication error, you may need to [set up the]({{site.url}}/docs/enterprise-registry/initial-setup/) `.dockercfg` file again.
+<img src="enable-build.png" class="img-center" alt="Enable Dockerfile Build"/>
 
-To apply the new configuration, restart the container running the registry.
+- Click the gear icon (<img src="gear.png">) and scroll down to the section entitled <strong> Dockerfile Build Support</strong>.
```

Comment:
What about `Click the configuration tab (image) and scroll` so we subtly teach that its about configuration.

Also, I bet it would look better and be simpler if we just use a Font Awesome gear in the doc:

`<span class="fa fa-gear"></span>`

It should render properly on our site at least. Any probably on GitHub too, right?